apps/flair/lib/flair/durations/init.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Flair.Durations.Init do
+  @moduledoc false
+
+  use Task, restart: :transient
+
+  @topic_reader Application.get_env(:flair, :topic_reader)
+
+  def start_link(_) do
+    Task.start_link(__MODULE__, :run, [])
+  end
+
+  def run do
+    reader = bootstrap_reader()
+    writer = Flair.Durations.Consumer.bootstrap()
+
+    ^reader = writer
```

Comment:
these are both `:ok` or exception? interesting...

Reply:
Yeah, and it returns `:ok` if both are `:ok`. In an intermediary commit, I had a complicated `with` statement that I really didn't like. This ended up being the simplest thing that I could think of.

Reply:
I think an explanatory comment would be nice unless you're planning on immediately refactoring.